openedx/core/djangoapps/edx_global_analytics/tasks.py

```diff
@@ -80,21 +79,20 @@ def collect_stats():
 
     olga_settings = settings.ENV_TOKENS.get('OPENEDX_LEARNERS_GLOBAL_ANALYTICS')
 
-    try:
-        token_object = TokenStorage.objects.first()
-        secret_token = token_object.secret_token
-    except AttributeError:
-        secret_token = ""
+    olga_acceptor_url = \
+        olga_settings.get(
+            'OLGA_ACCEPTOR_PERIODIC_TASK_URL'
+        ) or \
+        olga_settings.get(
+            'OLGA_ACCEPTOR_PERIODIC_TASK_URL_LOCAL_DEV'
+        )
 
-    platform_url = "https://" + settings.SITE_NAME
-
-    post_url = \
-        olga_settings.get('OLGA_PERIODIC_TASK_POST_URL') or olga_settings.get('OLGA_PERIODIC_TASK_POST_URL_LOCAL')
-
-    if not post_url:
+    if not olga_acceptor_url:
```

Comment:
for example, one for preparing data, second for sending data, one for authentication and main for control flow. Or another way.